website/project/views/node.py

```diff
@@ -586,20 +543,35 @@ def togglewatch_post(**kwargs):
         'watched': user.is_watching(node)
     }
 
+@must_be_logged_in
+@must_be_valid_project
+@must_not_be_registration
+@must_have_permission(ADMIN)
+def update_node(auth, node, **kwargs):
+    try:
+        return {
+            'updated_fields': {
+                key: getattr(node, key)
+                for key in
+                node.update(js_to_python(request.json))
```

Comment:
In APIv2, fields will use snake case (for both input and output). I think we can do the same here so we don't have to do this inflection.